### PR TITLE
Create Contact section 

### PR DESCRIPTION
**Pull request by youmari**
## In this branch this did the following : 
- Add a contact form with : 
     - Name field
     - Email field
     - Text area field
- Add validation to the fields
- Style the contact from a Figma design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -290,7 +290,46 @@ <h2>About me</h2>
...
+              required
+            />
+            <textarea
+              name="user-message"
+              id="user-message"
+              cols="30"
+              rows="5"
+              maxlength="200"
+              placeholder="f"
+            ></textarea>
```
Comment (zieeco) [inline index.html]: * [ ] Nice adding the `max-length` and `placeholder` but I think you're still missing something. 
The `max-length` is supposed to be able to accept 500 characters and your `placeholder` text is not meaningful. 
A user will be limited in terms of writing a message in your form as 200 characters is too small and also, a user won't understand what the letter `f` means in your `<textarea>`.
Please,  kindly do update them.

File: css/main.css [hunk 2]
```diff
@@ -325,3 +325,90 @@ li {
...
+}
+
+.footer .footer-header p {
+  text-align: center;
+  color: #172b4d;
+  font-size: 32px;
+  font-family: "Inter", sans-serif;
+  font-weight: bolder;
+  line-height: 42px;
+}
```
Comment (zieeco) [inline css/main.css:351]: * [ ] Great job on following the design from [Figma](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=48%3A838), but there are still some minor things you need to pay attention to. Please, refer to the images below for some improvement(s).


> || **YOUR DESIGN** || FIGMA DESIGN ||
> |-|---------|-|---------|-|
> ||   yordesign  || margin || 

- As you can see you need to add some space between your `.footer .footer-header p` and the first input in your form.
You can add `margin-bottom`. _There are many ways to achieve that._

File: css/main.css [hunk 3]
```diff
@@ -325,3 +325,90 @@ li {
...
+  font-size: 15px;
+  line-height: 24px;
+  font-weight: 400;
+  width: 65%;
+  box-sizing: border-box;
+  padding: 12px;
+  font-family: "Inter", sans-serif;
+  height: 114px;
+}
+
```
Comment (zieeco) [inline css/main.css:395]: * [ ] Great job on following the design from [Figma](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=48%3A838), but there are still other minor things that could improve your design Please, refer to the images below for some improvement(s).


> || **YOUR DESIGN** || FIGMA DESIGN ||
> |-|---------|-|---------|-|
> ||  width_2 || width || 

- As you can see your `textarea's width` is too small. You can play with it to fix the issue

File: css/main.css [hunk 4]
```diff
@@ -325,3 +325,90 @@ li {
...
+  font-size: 17px;
+  font-family: "Inter", sans-serif;
+  text-align: center;
+}
+
+.footer-form .see-project-button:hover {
+  color: #ff6b00;
+  background: #fff5e1;
+  border: 1px solid #f55800;
+}
```
Comment (zieeco) [inline css/main.css:414]: * [ ] You've done great here also but you're still missing some CSS-pseudo classes. 
- Add the `active`, `enabled`, and `disabled` pseudo-classes to your `.see-project-button` as this help to improve user experience.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @youmari , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Nice commits:heavy_check_mark: 
- Descriptive PR :heavy_check_mark: 
- Nice design :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
# Hi @youmari , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[Highly Recommended]
- _( optional )_ Please make sure to make README professional. This is a general requirement. I will highly recommend following the [microverse template](https://github.com/microverseinc/readme-template).


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**